daemon/src/maker_cfd.rs

```diff
@@ -691,20 +691,12 @@ where
                     tracing::warn!("Failed to handle rollover proposal V2: {:#}", e);
                 }
             }
-            wire::TakerToMaker::RolloverProtocol { .. } => {
-                unreachable!("This kind of message should be sent to the rollover_maker::Actor`")
-            }
-            wire::TakerToMaker::Protocol { .. } => {
-                unreachable!("This kind of message should be sent to the `setup_maker::Actor`")
-            }
-            wire::TakerToMaker::Hello(_) => {
-                unreachable!("The Hello message is not sent to the cfd actor")
-            }
-            wire::TakerToMaker::HelloV2 { .. } => {
-                unreachable!("The HelloV2 message is not sent to the cfd actor")
-            }
-            wire::TakerToMaker::Unknown => {
-                // Ignore unknown message to be forwards-compatible.
```

Comment:
why don't you want to be forwards-compatible??? you might have gone too far with combining them :)

Reply:
I think this comment was misleading. 
I think the decision to process / not process a message should *not* be taken in `maker_cfd` but in the connection handling actor, i.e. `maker_inc_connections`. That's why I moved it there and made all message that we don't care about in `maker_cfd` `unreachable`. 
Am I missing something?

Reply:
fair enough! I missed this, it makes sense to squash it there, instead of moving it to this actor.